jena/src/test/scala/org/w3/banana/jena/TestAnyRDFReader.scala

```diff
@@ -0,0 +1,50 @@
+package org.w3.banana.jena
+
+import org.scalatest.FunSuite
+import org.apache.log4j.Logger
+import org.w3.banana.AnyRDFReaderModule
+import org.w3.banana.RDFOpsModule
+import org.scalatest.FunSuite
+import org.w3.banana.TurtleWriterModule
+import java.io.StringWriter
+
+trait TestAnyRDFReader extends AnyRDFReaderModule with RDFOpsModule {
+  import ops._
+
+  val rdfxmlDocURL = "http://eulergui.sourceforge.net/doap_eulergui.rdf"
```

Comment:
Please no tests that depends on external resources. It's too easy to get failing tests because of that.